lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -538,82 +546,17 @@ withNetworkLayerBase tr np addrInfo (versionData, _) action = do
           where
             mkErr = ErrStakeDistributionQuery . T.pack . show
 
-        queryStakeDistribution pt = \case
-            AnyCardanoEra ShelleyEra -> do
-                let cmd = CmdQueryLocalState pt (QueryIfCurrentShelley Shelley.GetStakeDistribution)
-                result <- handleQueryResult "GetStakeDistribution"
```

Comment:
The tracing would need to be re-added when _interpreting_ the LSQ monad.

Reply:
https://github.com/input-output-hk/cardano-wallet/pull/2449#discussion_r559741621